scss/atoms/typography/_typography.scss

```diff
@@ -4,123 +4,4 @@
 
 @import
 	"variables",
-	"functions",
-	"mixins";
-
-body {
-	font: $normal #{ map-get( $root-font, font-size ) }/1.5 $sans-serif;
-
-	@media ( max-width: 800px ) {
-		font-size: 16px;
-	}
-	@media ( max-width: 500px ) {
-		font-size: 14px;
-	}
-}
-
-h1 {
```

Comment:
it shouldnt be there is some other whole files deleted that should not be as well not sure what happened there ill fix this and @arthurvr comments